tests/test_parameter_mapping.py

```diff
@@ -263,20 +262,20 @@ def test_all_override(condition_df_2_conditions,
         actual = petab.get_optimization_to_simulation_parameter_mapping(
             measurement_df=measurement_df,
             condition_df=condition_df,
-            sbml_model=sbml_model, parameter_df=parameter_df)
+            sbml_model=ss_model.model, parameter_df=parameter_df)
         assert actual == expected
 
     @staticmethod
-    def test_partial_override(condition_df_2_conditions,
-                              minimal_sbml_model):
+    def test_partial_override(condition_df_2_conditions):
         # Condition-specific parameters, keeping original parameters
         condition_df = condition_df_2_conditions
 
-        _, sbml_model = minimal_sbml_model
-        add_global_parameter(sbml_model, 'dynamicParameter1')
-        add_global_parameter(sbml_model, 'observableParameter1_obs1')
-        add_global_parameter(sbml_model, 'observableParameter2_obs1')
-        add_global_parameter(sbml_model, 'observableParameter1_obs2')
+        import simplesbml
+        ss_model = simplesbml.SbmlModel()
+        ss_model.addParameter('dynamicParameter1', 0.0)
+        ss_model.addParameter('observableParameter1_obs1', 0.0)
+        ss_model.addParameter('observableParameter2_obs1', 0.0)
+        ss_model.addParameter('observableParameter1_obs2', 0.0)
```

Comment:
These observable parameters were removed in `tests/test_observables.py::test_get_output_parameters`, but kept here?

Reply:
Nevermind, misread the other test